product-docs/features/mappings.md

```
@@ -0,0 +1,26 @@
+icon: diagram-sankey
+---
+
+# Mappings
+
+One common challenge when dealing with audits is reusing your assessment on one framework to move to a different one. This is commonly referred to as **mapping** or **crosswalk** between standards.
+
+CISO Assistant supports this capability and lets you create a projection of the content of an audit, given that a mapping is available.
+
+Mappings are library objects that can be customised, imported, and submitted to the community. To see the available ones, head to the libraries store and filter to mapping:
+
+![](<../.gitbook/assets/image (1) (1) (1) (1) (1) (1).png>)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)